backend/modules/simulation/router.py

```
@@ -1 +1,169 @@
+    except NotFoundError as e:
+        raise HTTPException(status_code=404, detail=str(e))
+    except ForbiddenError as e:
+        raise HTTPException(status_code=403, detail=str(e))
+    except Exception as e:
+        raise HTTPException(status_code=500, detail=str(e))
+
+
+@router.get("/scenes/{scene_id}", response_model=ScenarioSceneResponse)
+async def get_scene_by_id(
+    scene_id: int,
+    db: Session = Depends(get_db)
```

Ruff (0.14.8)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)